src/mono/wasm/runtime/rollup.config.js

```diff
@@ -48,13 +48,14 @@ const format = "iife";
 const name = "__dotnet_runtime";
 const inlineAssertQuotes = {
     // eslint-disable-next-line quotes
-    pattern: /assert\(([^,]*), *("[^"]*")\);/g,
-    replacement: "if (!($1)) throw new Error(`Assert failed: ${$2}`); // inlined assert"
+    pattern: /assert\(([^,]*), *"([^"]*)"\);/g,
```

Comment:
I really think this will bite us later. Let's give it a more unique name

Reply:
more unique name is good idea, maybe `mono_assert` ?
With that, the worst could happen is that it would not be matched and not inlined. 
We could add another dummier search for `mono_assert` as last step, if it can find some, it means the inlining didn't work for some reason and we could error on it.

Reply:
Yeah, also if it fails to inline in a hot path that will show up in the profiler tools, so that's not an issue. Just want to avoid it accidentally mangling something else due to the word assert